### PR TITLE
changes

### PR DESCRIPTION
**Pull request by mosesfawole**

### CODE REVIEW

File: about.html
Comment (ogeobubu) [inline about.html:46]: [factservices-43a43727c756d53119e306eccf7a03588d6c2650.zip](https://github.com/ogeobubu/factservices/files/5477173/factservices-43a43727c756d53119e306eccf7a03588d6c2650.zip)

**Review by ogeobubu — CHANGES_REQUESTED** [review]

**Review by mosesfawole — COMMENTED** [review]
...

File: about.html
Comment (ogeobubu) [reply about.html:46]: You've seen it right?

**Comment by mosesfawole** [comment]
i can't merge the pull request, its displaying this *Only those with write access to this repository can merge pull requests.*

**Comment by ogeobubu** [comment]
I just merged now

**Pull request merged**
